app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragmentViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.openevent.general.settings
+
+import android.arch.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import org.fossasia.openevent.general.auth.AuthService
+import timber.log.Timber
+
+class SettingsFragmentViewModel(private val authService: AuthService) : ViewModel() {
+
+    private val compositeDisposable = CompositeDisposable()
+
+    fun isLoggedIn() = authService.isLoggedIn()
```

Comment:
cant we use directyl authservice.isloggedin from activivty

Reply:
I tried that, I couldn't though , thats why made a view model